site/_posts/2017-09-04-the-school-year-begins.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Last Week in ManageIQ: The school year begins - let's improve our GPA"
+author: skateman
+date: 2017-09-05
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello, [Dávid](https://github.com/skateman) here from the UI team. Let's have a quick recap about what happened in the last week on the magical land of ManageIQ.
+
+Honestly, I gave up on counting how many repositories we are currently maintaining. The only number I come up with is ~220 and this is the number of PRs we merged last week. Around 50% of these PRs got merged into [core][manageiq PRs merged] and into the [two][manageiq-ui-classic PRs merged] [UI][[manageiq-ui-service PRs merged]] repos, feel free to check them out.
+
+## Featured
+[Martin](https://github.com/martinpovolny) (but not [Martin](https://github.com/himdel) and [Martin](https://github.com/hyperkid123)) decided to increase the [Code Climate GPA](https://codeclimate.com/github/ManageIQ/manageiq-ui-classic) of the UI Classic repo. I lost count of the PRs, but the outcome is [visible](https://codeclimate.com/github/ManageIQ/manageiq-ui-classic/trends/gpa).
+
+## Improved
+The Service UI guys hired a [new guy](https://github.com/apps/greenkeeper) for keeping their dependencies up to date and he is doing a [suspiciously good job](https://github.com/ManageIQ/manageiq-ui-service/pulls?q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222017-08-28+..+2017-09-03%22+sort%3Acreated-desc+author%3Aapp%2Fgreenkeeper). I am pretty sure that he will never lose his job because of the upcoming automatization and robots everywhere.
```

Comment:
Perhaps a gender-neutral "folks" or "people"/"person" would be better here?

Reply:
@imtayadeway thanks, there's no such thing as gender in my language so for me everything sounds like gender-neutral :wink: